lib/system_registry/task.ex

```diff
@@ -0,0 +1,52 @@
+defmodule SystemRegistry.Task do
+  @moduledoc """
+
+  Creates a process that executes a function anytime the value of a given
+  system_registry scope changes.
+
+  For example, lets say we want to perform a quick task any time the ipv4 address
```

Comment:
"For example, to perform an operation every time that the IPv4 address changes on "wlan0", do this:"